docs/application/dotnet/guides/text-input/voice-control-manager.md

```diff
@@ -0,0 +1,1084 @@
+
+3.  Prepare the voice control manager.
+
+    The preparation connects the voice control service for background work, such as recording and recognizing the user voice.
+
+    When the application initializes and prepares the voice control manager, the voice control service is invoked and connected for the background work. The service and the application communicate as server and client.
+
+4.  Get the recognized command result and ASR.
+
+    The recognized command result and ASR are sent through a registered callback.
+    You can receive the command matching result that the user uttered.
+    And if the registered command is duplicated or the user speaks multiple commands, the recognition result can contain multiple results. In this case, you can select one or reject in the `VoiceControlManagerClient.SetRecognizedCommandsSelectionDelegate()` callbacks
+
+5.  When no longer needed, unset the voice control manager.
+
+    You must disconnect the voice control service and deinitialize the voice control manager using the `Unprepare()` and `Deinitialize()` methods of the [Tizen.Uix.VoiceControlManager](https://samsung.github.io/TizenFX/latest/api/Tizen.Uix.VoiceControlManager.html) class.
+
+The following figure illustrates the voice control manager life-cycle:
+
+**Figure: Voice control manager life-cycle**
+
+![Voice control manager life-cycle](./media/csapi_vc_manager.png)
+
+The user controls the voice control manager service state. In a general scenario:
+1.  The user starts recording for recognition by using a voice control manager application, button, or voice trigger. If the start is successful, the voice control service state changes to `Recording`. For more information on service states, see [Tizen.Uix.VoiceControlManager.ServiceState](https://samsung.github.io/TizenFX/latest/api/Tizen.Uix.VoiceControlManager.ServiceState.html) enumration.
+2.  After recording is completed, the service state changes to `Processing` for recognition processing.
+3.  After recognition is completed, the service state changes to `Ready`.
+
+
+
+
+## Prerequisites
+
+To enable your application to use the voice control functionality:
+
+1.  To use the [Tizen.Uix.VoiceControlManager](https://samsung.github.io/TizenFX/latest/api/Tizen.Uix.VoiceControlManager.html) class, the application has to request permission by adding the following privileges to the `tizen-manifest.xml` file:
+
+    ```
+    <privileges>
+        <privilege>http://tizen.org/privilege/recorder</privilege>
+        <privilege>http://tizen.org/privilege/voicecontrol.manager</privilege>
+    </privileges>
+    ```
+
+2.  To use the methods and properties of the [Tizen.Uix.VoiceControlManager](https://samsung.github.io/TizenFX/latest/api/Tizen.Uix.VoiceControlManager.html) class, include it in your application:
```

Comment:
@rishu-samsung Is there any specific reason that you changed "include" with "add"?

If not how about keeping the original "include", as it is a commonly used expression under dotnet/guides as you can check in other pages.

Reply:
@rishu-samsung I thought you agreed to keep the original sentence for this line.. Is there anything that I misunderstood?

Reply:
> @rishu-samsung I thought you agreed to keep the original sentence for this line.. Is there anything that I misunderstood?

@mijin85cho Apologies for the same. Yes we need to keep the original sentence. This might have caused due to checking other comments. 

@sungraejo Kindly keep the original sentence only. Well, the line numbers I mentioned above here, comments are still need to be replied or incorporated. Kindly check the and do the needful.